portal/trigger_states/empro_states.py

```diff
@@ -92,10 +92,10 @@ def enter_user_trigger_critical_section(user_id):
     sm = EMPRO_state(ts)
     sm.begin_process()
     # Record the historical transformation via insert.
-    current_app.logger.debug(
-        "record state change to 'inprocess' from "
-        f"enter_user_trigger_critical_section({user_id})")
     ts.insert(from_copy=True)
+    current_app.logger.debug(
+        "persist-trigger_states-new from enter_user_trigger_critical_section()"
```

Comment:
>persist-trigger_states-new

Are the long "tokens" meant to be used as an easy way to search ELK fields, ie a workaround for exact matches? Might be worth adding a TODO to figure out ELK exact matches, but maybe you expect to cut this logging out later

Reply:
I believe you can do exact matches with the `field.keyword` selector in ELK - the "tokens" you refer to are just me trying to be consistent with spelling, so a grep will hit them all.  Yes, cutting this down later would be reasonable, as would tuning logging to not include debug.